cmd/service.go

```diff
@@ -31,6 +35,29 @@ const (
 	// Add new service requests here.
 )
 
+var serviceSignals = map[serviceSignal]string{
```

Comment:
better to simply have a switch case here - this global list of a map is never used anywhere.

Reply:
done